templates/artist/artist_detail.html

```diff
@@ -199,10 +223,56 @@ <h1 class="header-font">{{ artist.name }}</h1>
 
 {% block extrajs %}
     <script src="https://checkout.stripe.com/checkout.js"></script>
+    <script type="text/javascript" src="{% static "js/vendor/smooth-scroll.js" %}"></script>
     <script type="text/javascript">
+        var doughnutData = [
+                {
+                    value: {{ campaign.fans_percentage }},
+                    color: "#46BFBD",
+                    highlight: "#5AD3D1",
+                    label: "Fan %"
+                },
+                {
+                    value: {{ campaign.artist_percentage }},
+                    color:"#F7464A",
+                    highlight: "#FF5A5E",
+                    label: "{{ artist.name|escapejs }} %"
+                }
+
+            ];
+
+        var randomScalingFactor = function(){ return Math.round(Math.random()*100)};
+        var lineChartData = {
+            labels : ["1x","2x","3x","4x","5x"],
+            datasets : [
+                {
+                    label: "Artist Revenue",
+                    fillColor : "rgba(220,220,220,0.2)",
+                    strokeColor : "rgba(220,220,220,1)",
+                    pointColor : "rgba(220,220,220,1)",
+                    pointStrokeColor : "#fff",
+                    pointHighlightFill : "#fff",
+                    pointHighlightStroke : "rgba(220,220,220,1)",
+                    data : [{{ campaign|percentage_roi:100 }},{{ campaign|percentage_roi:200 }},{{ campaign|percentage_roi:300 }},{{ campaign|percentage_roi:400 }},{{ campaign|percentage_roi:500 }}]
+                },
+            ]
+
+        }
+
         var perdiem_fee = {{ PERDIEM_FEE }};
```

Comment:
Move these lines:
```
var perdiem_fee = {{ PERDIEM_FEE }};
var artist_name = '{{ artist.name|escapejs }}';
{% if campaign %}
```
above line 228: `var doughnutData = [`

That way the JS that renders the graphs only run when the artist has a campaign.